string name
int16 x
int16 y
int16 size
int16 det_conf
